src/CodeGen_C.cpp

```diff
@@ -108,6 +108,12 @@ inline float float_from_bits(uint32_t bits) {return reinterpret<float, uint32_t>
 template<typename T> T max(T a, T b) {if (a > b) return a; return b;}
 template<typename T> T min(T a, T b) {if (a < b) return a; return b;}
 
+template<typename A, typename B>
```

Comment:
(I'd actually be fine with requiring C++11 for consumers, since we require it so pervasively elsewhere, but that's a different issue...)